### ISSUE TITLE
Unset lost-column?

### ISSUE THREAD

**Issue by iest** [peterramsing/lost#150]
From what I can see there's no way just to undo any lost-columns that are added. Are there any plans to support this or shall I just stick to using `lost-column: 1/1`?

**Comment by corysimmons** [comment]
What does `lost-column: auto` do? I haven't used Lost in a while so I forget my own syntax. :dancer:

**Comment by corysimmons** [comment]
If `lost-column: 1/1` works for you then I'd just continue using that.

**Comment by iest** [comment]
I've ran into this again...

Had to resort to pretty horrid unsetting like so:

```css
  .links .spacer:nth-child(1n),
  .links .spacer:nth-child(1n + 1),
  .links .spacer:nth-child(n),
  .links .cta:nth-child(1n),
  .links .cta:nth-child(1n + 1),
  .links .cta:last-child,
  .links li:nth-child(n) {
    float: inherit;
    clear: inherit;
    margin-right: inherit;
    width: inherit;
  }
```

@corysimmons where should I get started if I wanted to implement `lost-column: auto`?

**Comment by sarith** [comment]
I'd rather see this implemented as `lost-column: none` or something.  We definitely need the ability to completely clear lost's added rules (for example, on different breakpoints to overwrite them).

**Comment by corysimmons** [comment]
Reopening. There should be a way to reset `lost-column`.

**Comment by peterramsing** [comment]
I'm with @sarith. A team I work with has been having to overwrite at different breakpoints as well. I've been playing with adding a `lost-column: none` rule and time permitting I'll see if I can wrap something into a pull request in the future.

**Comment by corysimmons** [comment]
@peterramsing Would really appreciate that PR. :)

**Comment by peterramsing** [comment]
@corysimmons I'm playing with a conditional at the top of the `lib/lost-column.js` file that looks for `none.

[Working copy of file Gist](https://gist.github.com/peterramsing/8ff973143033c18907c3)

```css
foo {
  lost-column: none;
}
```

The outputted CSS is:
```css
foo {
  width: inherit;
}
foo:last-child {
  float: inherit;
  clear: inherit;
  margin-right: inherit;
  width: inherit;
}
foo:nth-child(n) {
  float: inherit;
  clear: inherit;
  margin-right: inherit;
  width: inherit;
}
foo:nth-child(1n + 1) {
  float: inherit;
  clear: inherit;
  margin-right: inherit;
  width: inherit;
}
foo:nth-child(1n) {
  float: inherit;
  clear: inherit;
  margin-right: inherit;
  width: inherit;
}
```

I want to make it a bit more intelligent than that I think, but maybe it could be that simple. It was a quick hack I threw in on a flight so I'm still testing it but wanted your thoughts on if it could indeed be that simple since you know the codebase a lot better than I.

### Regarding Sytax
Also, I don't want to just assume the syntax should be "none" to clear. Maybe this is a `lost-utility` instead of a part of `lost-column`. What do you think for direction. Maybe `unset` might be a better option.

**Comment by peterramsing** [comment]
@corysimmons We're running into this again in our codebase again and I'd love do see if we could implement this feature. Have any feedback?

**Comment by peterramsing** [comment]
Tested this more and the current version is working well but @corysimmons –let me know if this feels too tacky. I'll get a PR going so we can discuss this more there.

**Comment by corysimmons** [comment]
How about `lost-column: reset`? I'm also happy with `none`.

**Comment by peterramsing** [comment]
What about "inherit"?

**Comment by peterramsing** [comment]
We've got a thread going in my office about what it could be named. @iest @sarith Thoughts?

**Comment by peterramsing** [comment]
"unset"? https://developer.mozilla.org/en-US/docs/Web/CSS/unset

**Comment by iest** [comment]
I like `none` — it's a float/flex layout, and both `float` and `flex` have `none` to undo any values.

**Comment by corysimmons** [comment]
I think I agree. I like `none` as well.

**Comment by peterramsing** [comment]
`none` it is.

**Comment by peterramsing** [comment]
This is addressed with #211.

**Comment by corysimmons** [comment]
Moving discussion over there. This issue can probably be closed since it's in the works.
